docker/docker-compose.yml

```diff
@@ -138,7 +138,7 @@ services:
       JPDA_TRANSPORT: dt_socket
     entrypoint: ["catalina.sh", "jpda", "run"]
     ports:
-      - "127.0.0.1:5000-5009:5000"
```

Comment:
I'm fine with this change, but just wondering if you meant to check this one in?